packages/react-ui/src/theme/carto-theme.js

```diff
@@ -58,7 +58,7 @@ export const cartoThemeOptions = {
     /* Custom Colors palette */
     qualitative: { ...commonPalette.qualitative },
     default: { ...commonPalette.default },
-    charts: { ...componentsPalette.charts },
+    cartoBrand: { ...commonPalette.cartoBrand },
```

Comment:
The name looks a bit strange to me... saying 'cartoBrand' inside a carto theme seems a bit redundant, how about just brand?

Reply:
Fine for me. I've just leave a comment on Figma suggesting this change on their side too, to use de same naming.

Reply:
Done, they already did the change on their side too